src/guides/v2.4/release-notes/open-source-2-4-2.md

```diff
@@ -1098,6 +1614,18 @@ We have fixed hundreds of issues in the Magento 2.4.2 core code.
 
 *  Redundant less styles navigation and unused properties in the Blank theme have been removed. [GitHub-29914](https://github.com/magento/magento2/issues/29914)
 
+<!-- ENGCOM-8285 -->
+
+*  Added collapsible navigation functionality to the Blank theme in mobile view. [GitHub-30237](https://github.com/magento/magento2/issues/30237)
+
+<!-- ENGCOM-7803 -->
+
+*  Web font loading for themes has been optimized.  [GitHub-29526](https://github.com/magento/magento2/issues/29526)
+
+<!-- ENGCOM-8419 -->
+
+*  Deployment of specific themes for multiple locales has been optimized, which improves deployment that use a single theme with different locales. [GitHub-30184](https://github.com/magento/magento2/issues/30184)
```

Comment:
The description of the issue no longer reflects what actually was approved.
So here is what I suggest should be used as a more accurate description:

> A new `--no-parent` flag was added to the `bin/magento setup:static-content:deploy` command which prevents the parent(s) of a theme to be compiled (if any). Often this was unnecessary and with this new flag you can significantly improve the performance of the static content deployment process. However, be aware that this new flag does not work when using the `compact` strategy.

Reply:
thanks for the update -- edited and added to the notes.